src/modules/folder/folder.usecase.ts

```diff
@@ -30,6 +30,7 @@ import { WorkspaceAttributes } from '../workspaces/attributes/workspace.attribut
 import { FileUseCases } from '../file/file.usecase';
 import { File, FileStatus } from '../file/file.domain';
 import { CreateFolderDto } from './dto/create-folder.dto';
+import { Transaction } from 'sequelize';
```

Comment:
Also, adding database-related things directly here breaks the decoupling the repositories achieve, as this is coupling the use cases to the underlying technology to persist data. Therefore, this is completely discouraged.